src/main/java/im/status/keycard/KeycardApplet.java

```diff
@@ -116,6 +116,9 @@
   static final byte SFLAG_EXPORTABLE = 1;
   static final byte SFLAG_MAX = 1;
 
+  static final byte INIT_P1_FIRST = 0x00;
+  static final byte INIT_P1_NEW = 0x01;
```

Comment:
Might be better to be more descriptive with the INIT_P1_NEW name; suggested: INIT_P1_SET_NEW_PAIRKEY?

Reply:
Updated